### ISSUE TITLE
What is the best way to auto classify unknown faces?

### ISSUE THREAD

**Issue by mbektimirov** [cmusatyalab/openface#93]
Hi,
Is it possible to train an SVM model without predefined names or classes by classifying faces with automatic ids? What I'm trying to solve is
- detect multiple faces with `align.getAllFaceBoundingBoxes(...)` on images with unknown people
- train the classifier with the detected data and save to db or model file
- feed another faces to the classifier and find them in db by autogenerated class id

The problem is that multiple faces in one frame can move and shuffle (training on video stream). I found [auto-sklearn](http://auto-sklearn.readthedocs.org/) but it needs classes from the start too.

**Comment by bamos** [comment]
Hi @mbektimirov - maybe you can cluster the faces of unknown people with something like http://scikit-learn.org/stable/modules/generated/sklearn.cluster.Birch.html? I'm closing this issue since the GitHub issue tracker is for development discussions and bug tracking. Please follow up on our mailing list or chatroom if you want to further discuss this.

-Brandon.
